mempool/src/shared_mempool/tasks.rs

```diff
@@ -357,6 +358,88 @@ where
     statuses
 }
 
+/// Submits a list of SignedTransaction to the local mempool
+/// and returns a vector containing SubmissionStatusBundle.
```

Comment:
nit: [SubmissionStatusBundle] to be consistent

Reply:
But also, should we add comments on how the perf test implementation differs from prod, on all of the `#[cfg(feature = "consensus-only-perf-test")]` functions?

Reply:
Sure, I have added as much info as possible. Let me know if something is missing.